flink-autoscaler/src/main/java/org/apache/flink/autoscaler/metrics/ScalingMetrics.java

```diff
@@ -217,14 +217,13 @@ private static double getNumRecordsInPerSecond(
             JobVertexID jobVertexID,
             boolean isSource) {
         var numRecordsInPerSecond = flinkMetrics.get(FlinkMetric.NUM_RECORDS_IN_PER_SEC);
-        if (isSource && (numRecordsInPerSecond == null || numRecordsInPerSecond.getSum() == 0)) {
-            numRecordsInPerSecond =
-                    flinkMetrics.get(FlinkMetric.SOURCE_TASK_NUM_RECORDS_IN_PER_SEC);
-        }
```

Comment:
Why the `SOURCE_TASK_NUM_RECORDS_IN_PER_SEC` part is removed?

Reply:
@1996fanrui  Sry, I have put it back and just add the comments